lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { shopIdAutoValue } from "./helpers";
+import { Address } from "./address";
+import { Metafield } from "./metafield";
+
+/**
+ * Accounts Schemas
+ */
+
+export const Profile = new SimpleSchema({
```

Comment:
Like add the `const Profile` to [`/lib/schemas.js`](https://github.com/mikeumus/reaction-example-plugin/blob/3ba6d468388d7d3a6262f6072572c7973b411b92/lib/schemas.js) and import it from there?